EssentialFeed/EssentialFeedTests/Image Comment/Presentation/ImageCommentsPresenterTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsPresenterTests: XCTestCase {
+	func test_title_isLocalized() {
+		XCTAssertEqual(ImageCommentsPresenter.title, localized("IMAGE_COMMENTS_TITLE"))
+	}
+
+	func test_map_createsViewModel() {
+		let now = Date()
+		let calendar = Calendar.current
```

Comment:
You can also use a specific calendar to decouple the test from global environment details.

```suggestion
		let calendar = Calendar(identifier: .gregorian)
```